music21/configure.py

```diff
@@ -44,11 +44,13 @@
 reFinaleReaderApp = re.compile(r'Finale Reader\.app', IGNORECASE)
 reMuseScoreApp = re.compile(r'MuseScore.*\.app', IGNORECASE)
 reMuseScoreExe = re.compile(r'Musescore.*\\bin\\MuseScore.*\.exe', IGNORECASE)
+reDoricoApp = re.compile(r'Dorico.*\.app', IGNORECASE)
+reDoricoExe = re.compile(r'Dorico.*\.exe', IGNORECASE)
 
-urlMusic21 = 'https://web.mit.edu/music21'
+urlMusic21 = 'http://web.mit.edu/music21'
```

Comment:
I'm curious why this unrelated change? Sphinx `linkcheck` will emit errors for redirects from http -> https, so we just link to https.

Reply:
I did not (Intentionally) make any such change.